qiskit_experiments/library/randomized_benchmarking/rb_experiment.py

```diff
@@ -206,284 +233,136 @@ def _sequences_to_circuits(
             ):
                 prev_elem, prev_seq = self.__identity_clifford(), []
 
-            qubits = list(range(self.num_qubits))
             circ = QuantumCircuit(self.num_qubits)
-            circ.barrier(qubits)
+            circ.append(Barrier(self.num_qubits), circ.qubits)
```

Comment:
Using `_append` makes code slightly faster by bypassing the validation. Do you have reason to stick to validation?

Reply:
This comment is not addressed. Of course you can do this in the follow-up.